samples/java-protobuf-tracing/README.md

```diff
@@ -0,0 +1,60 @@
+## This example show how to create Spans by the users
+ 
+
+
+## Running Locally
+
+
+When running a Kalix service locally, we need to have its companion Kalix Runtime running alongside it.
+
+To start your service locally, run:
+
+```shell
+mvn kalix:runAll
+```
+
+It's worth noting that `application.conf` is passing `kalix.telemetry.tracing.collector-endpoint="http://localhost:4317"` 
+to the application so the SDK knows where to export the traces. This is NOT needed when deploying in Kalix, only when run in local, that is, `mvn kalix:runAll`.
+
+This command will start your Kalix service and a companion Kalix Runtime as configured in [docker-compose.yml](./docker-compose.yml) file.
+This will also start a Jaeger service to which the services above will push the traces. You can find Jaeger at `http://localhost:16686`
+
+
+With both the Kalix Runtime and your service running, any defined endpoints should be available at `http://localhost:9000`. In addition to the defined gRPC interface, each method has a corresponding HTTP endpoint. Unless configured otherwise (see [Transcoding HTTP](https://docs.kalix.io/java-protobuf/writing-grpc-descriptors-protobuf.html#_transcoding_http)), this endpoint accepts POST requests at the path `/[package].[entity name]/[method]`.
+For example, using [`grpcurl`](https://github.com/fullstorydev/grpcurl):
+
+```shell
+grpcurl -plaintext localhost:9000  com.example.Controller/CallAsyncEndpoint                                 
+```
+produces
+```
+{
+  "message": "{\n  \"userId\": 1,\n  \"id\": 1,\n  \"title\": \"sunt aut facere repellat provident occaecati excepturi optio reprehenderit\",\n  \"body\": \"quia et suscipit\\nsuscipit recusandae consequuntur expedita et cum\\nreprehenderit molestiae ut ut quas totam\\nnostrum rerum est autem sunt rem eveniet architecto\"\n}"
+}
+```
+or 
```

Comment:
This `or` looks strange. 
The next command and output is exactly the same as the first. 

Where you planning to add a REST example here or maybe the `CallSyncEndpoint`?

Reply:
removed.
Yes but as we removed the `Sync` one it's the `or` also removed.